build_data.js

```diff
@@ -26,7 +26,7 @@ const request = require('request').defaults({ maxSockets: 1 });
 let _currBuild = null;
 
 
-function buildData() {
+function buildData(build_type) {
```

Comment:
the 'BUILD_TYPE' environment variable can be overridden in the package.json file in one of the new 'internal' build rules.